tests/postgres_tests/test_citext.py

```diff
@@ -6,25 +6,35 @@
 from django.db import IntegrityError
 
 from . import PostgreSQLTestCase
-from .models import CITextTestModel
+from .models import CITestModel
 
 
 class CITextTestCase(PostgreSQLTestCase):
 
     @classmethod
     def setUpTestData(cls):
-        CITextTestModel.objects.create(name='JoHn')
+        CITestModel.objects.create(
+            name='JoHn',
+            email='joHn@johN.com',
+            description='Just an Average Joe named JoHn',
+        )
 
-    def test_equal_lowercase(self):
+    def test_equal_lowercase_cicharfield(self):
         """
         citext removes the need for iexact as the index is case-insensitive.
         """
-        self.assertEqual(CITextTestModel.objects.filter(name='john').count(), 1)
+        self.assertEqual(CITestModel.objects.filter(name='john').count(), 1)
 
-    def test_fail_case(self):
+    def test_fail_citext_primary_key(self):
         """
-        Creating an entry for a citext-field which clashes with an existing
-        value isn't allowed.
+        Creating an entry for a citext-field used as a primary key which
+        clashes with an existing value isn't allowed.
         """
         with self.assertRaises(IntegrityError):
-            CITextTestModel.objects.create(name='John')
+            CITestModel.objects.create(name='John')
+
+    def test_equal_ciemailfield(self):
+        self.assertEqual(CITestModel.objects.filter(email='john@john.com').count(), 1)
+
+    def test_equal_citextfield(self):
+        self.assertEqual(CITestModel.objects.filter(description='Just an Average Joe named JoHn').count(), 1)
```

Comment:
The description should be lower cased. I 'm not sure if creating separate test methods for each field is adding much value. You could use `subTest()` or just three assertions in a single test.